include/glow/Base/Tensor.h

```diff
@@ -850,7 +850,7 @@ template <class ElemTy> class Handle final {
     assert(low >= std::numeric_limits<ElemTy>::lowest() &&
            high <= std::numeric_limits<ElemTy>::max() &&
            "Cannot initialize outside range of representable values.");
-    std::uniform_int_distribution<int> dist(low, high);
+    std::uniform_int_distribution<int long> dist(low, high);
```

Comment:
int long ? what does it do? :D

long long?

Reply:
long long, changed.

Reply:
I've just never seen it. Interesting that it worked.

Reply:
agreed :P